### PR TITLE
Patch Release 2.5.2

### PR DESCRIPTION
**Pull request by awhitford**
Note that the README had some whitespace removed thanks to the _Format on Save_ setting on my editor. There are very few actual changes.

### CODE REVIEW

**Review by aagarwal1012 — APPROVED** [review]
lgtm

**Comment by awhitford** [comment]
@aagarwal1012 I think you need to add tags for my patches to appear under **Releases** on GitHub.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div align="center"><img src="https://github.com/aagarwal1012/Animated-Text-Kit/blob/master/display/cover.gif?raw=true"/></div>  
+<div align="center"><img src="display/cover.gif?raw=true"/></div>
```
Comment (aagarwal1012) [inline README.md:1]: @awhitford, these relative paths actually don't work on https://pub.dev/packages/animated_text_kit, please revert these changes and create a new release.

**Comment by aagarwal1012** [comment]
@awhitford, I will create a new release after you fix the Readme issue.

File: README.md (hunk 1 above)
Comment (awhitford) [reply README.md:1]: Ugh!  I was not thinking of `pub.dev` -- I made the change because from GitHub's perspective, it makes validating a branch more accurate -- otherwise it can get confusing when you make a change on a branch, but this still points to master. However, I get that `pub.dev` is more important. I'm sorry.
